src/compliance.cc

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023 Google LLC
+// SPDX-License-Identifier: BSD-2-Clause
+
+#if defined(AVIF_ENABLE_COMPLIANCE_WARDEN)
+
+#include <cstddef>
+#include <cstdint>
+#include <limits>
+
+#include "avif/internal.h"
+#include "box_reader_impl.h"
+#include "spec.h"
```

Comment:
Maybe add a comment that this is in ComplianceWarden/src/utils/ ?
otheriwse I find bare filenames like that quite mysterious, but I'm not sure they can be improved, unless you have all of the ext/ directory in the includes path